src/commands/shell.js

```diff
@@ -0,0 +1,38 @@
+const { requireHerbarium } = require('../generators/utils')
+const repl = require('@herbsjs/herbs2repl')
+
+const cmd = {
+  name: 'shell',
+  description: 'Run Herbs Shell',
+  run: async toolbox => {
+    const { red } = toolbox.print.colors
+    const herbarium = requireHerbarium("shell", toolbox.filesystem.cwd())
+
+    if (!herbarium.requireAll) {
+      toolbox.print.info(`\n ${red('• Exit with error: Herbarium not found 😢')}`)
+      return
+    }
+
+    herbarium.requireAll()
+
+    async function main(injection) {
+      const usecases = Array.from(herbarium.usecases.all).map(([_, item]) =>
+          ({ usecase: item.usecase(injection), id: item.id, tags: { group: item.group || "useCase" } }))
+
+      const user = {
```

Comment:
@danielsous `user` is the authorization setting, right? This can be a parameter to the `shell` command. What do you think?

Reply:
Good, I will add the parameters to this command

Reply:
Good point. Probably the paramter should be ready to accept a Json string, something like `--user:'{ "canDoIt": "true" }'` or a simple string like `--user:admin`

Reply:
I like something more along those lines.

![image](https://user-images.githubusercontent.com/50839854/177550029-42da2c91-b6f8-41fb-8383-5162cedd1a57.png)

![image](https://user-images.githubusercontent.com/50839854/177550108-934f540a-ad21-478e-8a06-b148efe9638c.png)

Reply:
I don't know if I understood correctly, but the approach seems interesting.

I liked the idea of bringing the user's choice to the REPL, but I don't know if it would be possible to bring the permissions for the user to choose because each system will have different formats.

what we might have is a file with the list of users and their default permissions `infra\shell\users.js`:

```js

module.exports = {
	Admin: {
		canCreateUser: true,
		...
	},
	Client: {
		canCreateUser: false,
		...
	},
	User: () => ({ canCreateUser: false }),
}
```